### DISCUSSION TITLE
Question: How to Get HDMI when using DVI adapter

### DISCUSSION THREAD

**Discussion by jtitley** [macOS-Dell-Latitude-E7470/macOS-Dell-Latitude-E7470#17] [Q&A]
Anyone knows how to make DVI adapter work. I confirmed HDMI straight works, but my monitor is DVI. Weird that my other hackintosh's don't have this problem.

**Comment by adityabakare** [comment]
Since we don't have a dedicated DVI port, I don't feel like I should be flagging this as an issue. Moreover, as this is a question, I'm moving this to Discussions for others to discuss and try to resolve any doubts.

Edit: I can try and add some missing properties of WEG in the next update, but I can't guarantee if that will work.

**Reply by Aristocrat-jt** [reply]
I agree, this shouldn't be an issue, but a question. Because of the current situation I can't get a new HDMI monitor for a while, so keen to get this to work!

BTW, this has got to be the best hackintosh laptop profile and want to move this as my primary machine.

Happy to test or if you can point me to how to figure it out, I'm willing to do so. I noticed there are references to hackintool and framebuffer mapping, but it didn't work. Or maybe I just wasn't doing it right.

**Reply by adityabakare** [reply]
@Aristocrat-jt Hackintool is known to break stuff a lot of times(at least for me). Therefore, I'll need to refer to the WEG manual again and see if I'm missing something. I'll link it over here, just in case if you can try and report back.

https://github.com/acidanthera/WhateverGreen/blob/master/Manual/FAQ.IntelHD.en.md

Hope this helps! :)

Edit:
In config.plist, under DeviceProperties > Add, try replacing the first dictionary with the code I'm attaching below, and report back if it fixes the problem. Make sure you keep the indentation intact, as it was originally in the file.

https://del.dog/lignexagag.txt

**Comment by adityabakare** [comment]
I've updated some framebuffer patches. Try the latest version and do report of your DVI adapter works for you.

**Reply by Aristocrat-jt** [reply]
Well, that worked, but once. The monitor properly detects. But, the laptop monitor goes blank and I don't know why. If the laptop sleeps, the monitor doesn't recover, but then the laptop screen is on. After that, no amount of plugging or unplugging of the HDMI/DVI cable will activate the monitor. Even power on, power off.

Will try your sleep suggestion as mentioned in my HDMI discussion before I blow away the laptop.
